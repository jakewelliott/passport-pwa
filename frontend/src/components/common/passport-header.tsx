export const PassportHeader = () => {
    return (
        <div className='flex flex-col items-center justify-center h-2/4'>
            <img src="./DPRLogoWhite.svg" alt="North Carolina Department of Parks and Rec White Logo" width="136" height="103" />
            <h3 className="text-center">North Carolina State Parks</h3>
            <p className='script h-20' style={{ paddingTop: 30 }}>Passport</p>
        </div>
    );
}
