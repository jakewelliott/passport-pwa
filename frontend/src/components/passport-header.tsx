export const PassportHeader = () => {
    return (
        <div className='flex h-2/4 flex-col items-center justify-center'>
            <img
                src='/DPRLogoWhite.svg'
                alt='North Carolina Department of Parks and Rec White Logo'
                width='136'
                height='103'
            />
            <h3 className='text-center'>North Carolina State Parks</h3>
            <p className='font-Norican text-8xl' style={{ paddingTop: 30 }}>
                Passport
            </p>
        </div>
    );
};
