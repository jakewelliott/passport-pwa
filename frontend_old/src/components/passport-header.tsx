export const PassportHeader = () => {
    return (
        <div className='flex h-2/4 flex-col items-center justify-center'>
            <img
                src='/DPRLogoWhite.svg'
                alt='North Carolina Department of Parks and Rec White Logo'
                width='136'
                height='103'
            />
            <h3 className='text-center'>North Carolina State Parks</h3>
            <p className='script h-20' style={{ paddingTop: 30 }}>
                Passport
            </p>
        </div>
    );
};
