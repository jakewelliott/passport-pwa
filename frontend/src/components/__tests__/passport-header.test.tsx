import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import { PassportHeader } from '../passport-header';

describe('PassportHeader', () => {
    it('matches snapshot', () => {
        const { container } = render(<PassportHeader />);
        expect(container).toMatchSnapshot();
    });
});
