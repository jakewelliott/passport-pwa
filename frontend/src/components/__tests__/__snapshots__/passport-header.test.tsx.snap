// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`PassportHeader > matches snapshot 1`] = `
<div>
  <div
    class="flex h-2/4 flex-col items-center justify-center"
  >
    <img
      alt="North Carolina Department of Parks and Rec White Logo"
      height="103"
      src="/DPRLogoWhite.svg"
      width="136"
    />
    <h3
      class="text-center"
    >
      North Carolina State Parks
    </h3>
    <p
      class="font-Norican text-8xl"
      style="padding-top: 30px;"
    >
      Passport
    </p>
  </div>
</div>
`;
